flocker/acceptance/endtoend/test_dockerplugin.py

```diff
@@ -118,7 +123,7 @@ def run_python_container(self, cluster, address, docker_arguments, script,
             self.addCleanup(client.remove_container, cid, force=True)
         return cid
 
-    def _create_volume(self, client, name, driver_opts):
+    def _create_volume(self, client, name, driver_opts, cleanup=True):
```

Comment:
it was initially, but thought it could be useful in the future. Will remove since nothing would use it. tx!